contracts/0.4.24/lib/StakeLimitUtils.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.4.24;
+
+//
+// We need to pack four variables into the same 256bit-wide storage slot
+// to lower the costs per each staking request.
+//
+// As a result, slot's memory aligned as follows:
+//
+// LSB ------------------------------------------------------------------------------> MSB
+// 0______________________32______________128_________________________160______________256
+// |______________________|________________|___________________________|________________|
+// | prevStakeBlockNumber | prevStakeLimit | maxStakeLimitGrowthBlocks | maxStakeLimit  |
```

Comment:
From the code it seems that fields are actually stored in the inverse order?

Reply:
Well, I believe it's not. 
LSB and MSB markers are here to disambiguate since EVM is big-endian under the hood.

Reply:
ah, I see

Reply:
maybe, let's still invert the direction in the doc, keeping LSB and MSB markers? it's much easier to read the code that follows in this way as shift operator symbols don't consider endianness

Reply:
btw, big-endian systems store LSB at the largest memory address, so this layout is incorrect anyway

Reply:
I also think that MSB and LSB needs to be changed here.

Reply:
I will update the sketch to make it more readable :+1: